src/20241201_Multi_IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances/IncreaseBorrowSlope1ToAllStablecoinsAcrossAllAaveInstances.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Increase Borrow Slope1 to all Stablecoins across all Aave Instances"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-increase-borrow-slope1-to-all-stablecoins-across-all-aave-instances/19979"
+---
+
+## Simple Summary
+
+This proposal recommends increasing the **borrow slope1** parameter for stablecoins across all Aave V3 Instances to **9.5%**. This adjustment aims to align interest rate models with the updated **Stable Debt Reserve (SDR)** strategy, promoting sustainability, better risk management, and optimal utilization of the protocol’s resources.
+
+## Motivation
+
+The current market environment presents a unique opportunity to optimize Aave’s interest rate parameters. With the introduction of sUSDe as E-Mode collateral and general market sentiment leading to higher desire for borrowing, we are seeing significantly higher utilisation in the stablecoin market. This adjustment to the borrow slope1 will better align Aave’s interest rates with prevailing market conditions, potentially leading to:
+
+- Increased protocol revenue through better rate optimization
+- Enhanced capital efficiency by maintaining competitive lending rates at lower volatility.
+
+Additionally, this change will ensure appropriate risk compensation for stablecoin lenders.
+
+## Specification
+
+Increase the **borrow slope1** parameter for all Stablecoin assets on Aave V3 Instances from its current value to **9.5%.**
+
```

Comment:
I Think, given the amount of changes, it would be more clear to have the final table here of all the changes